spring-cloud-tencent-examples/polaris-circuitbreaker-example/polaris-circuitbreaker-example-b2/src/main/resources/bootstrap.yml

```diff
@@ -11,3 +11,9 @@ spring:
       stat:
         enabled: true
         port: 28083
+#    tencent:
```

Comment:
The configuration should be placed in `polaris-circuitbreaker-example-a` ?

Reply:
fixed .